infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -27,7 +27,9 @@ public AbstractServer() {
      */
     public abstract CorfuMsgHandler getHandler();
 
-    public abstract boolean isServerReady();
+    public boolean isServerReadyToHandleMsg(CorfuMsg msg) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "msg". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)